sql/migrations/20230324015015_world.sql

```diff
@@ -0,0 +1,1644 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20230324015015');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20230324015015');
+-- Add your query below.
+
+TRUNCATE TABLE game_tele;
+INSERT INTO `game_tele` (`id`, `position_x`, `position_y`, `position_z`, `orientation`, `map`, `name`) VALUES
+(1, -5195.53, -2802.04, -7.71064, 0, 1, '1000cave'),
+(2, -4687, -1819, -57, 0, 1, '1000Needles'),
+(3, -8910.07, -192.197, 81.9402, 0, 0, 'abbeyhall'),
+(4, -8876.89, -176.531, 81.952, 0, 0, 'abbeylibrary'),
+(5, -8888.44, -167.285, 81.9391, 0, 0, 'abbeymain'),
+(6, -13662.8, -291.11, 8.17598, 0, 0, 'Aboraz'),
+(7, -7806.59, -3065.18, 56.5261, 6.15101, 1, 'abyssalsands'),
+(8, -11053.8, 226.013, 27.3777, 0, 0, 'addle'),
+(9, 248.003, -2146.14, 118.356, 0, 0, 'Aerie'),
+(10, 3044.74, 657.349, 75.3529, 0, 0, 'agacrypt'),
+(11, -1719.14, -1746.34, 95.083, 3.96389, 1, 'agama'),
+(12, -7045.2, -3322.84, 238.293, 0, 0, 'Agmondsend'),
+(13, 433.952, -3360.79, 119.676, 0.700589, 0, 'agolwatha'),
+(14, -9750.16, 1545.36, 42.9697, 3.14421, 1, 'ahn'),
+(15, -8137.54, 1526.96, 6.73688, 3.17847, 1, 'AhnQiraj'),
```

Comment:
Don't know if you want to adjust the PR or anything, but I would like to point out that a few `tele` locations have been edited between PTR builds/removed from the build these are from. Here are the specific differing entries I could find:
```sql
(15, -8488,    1513,    2, (No orientation included), 1, 'Ahnqiraj'), -- Build 4714
(15, -8137.54, 1526.96, 6.73688, 3.17847,             1, 'AhnQiraj'), -- Build 5496

(ID, -1156.79, 1900.2, 88.8571, 261.09, 1, 'ghost'), -- Build 4714
-- Not included in Build 5496

(757, -10947, -1968,    117, (No orientation included), 0, 'Karazhan'), -- Build 4714
(757, -11133, -2021.36, 47.4563, 0.668559,              0, 'Karazhan'), -- Build 5496
```

Reply:
Correction: apparently the `ghost` worldport was renamed in build 5496 to `GhostWalker` (which is already included in this PR), it was not removed outright as my first comment indicates.